braindecode/models/eegdeformer.py

```diff
@@ -0,0 +1,372 @@
+"""
+CBCR License 1.0
+
+Copyright 2022 Centre for Brain Computing Research (CBCR)
+
```

Comment:
I don’t think the license is compatible with our bsd license

Reply:
Hmm, okay, thank you for catching this. I considered attributing only this code to this particular license, but I think it would not be possible. I was benchmarking the model a little, and the performance is not super good for BCI

Reply:
@bruAristimunha can you share the (preliminary) results?